phoenix-flume-base/pom.xml

```diff
@@ -157,10 +168,37 @@
 
   <build>
     <plugins>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>build-helper-maven-plugin</artifactId>
-      </plugin>
+        <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>build-helper-maven-plugin</artifactId>
+            <version>3.0.0</version>
+            <executions>
+                <execution>
+                    <id>add-source</id>
+                    <phase>generate-sources</phase>
+                    <goals>
+                        <goal>add-source</goal>
+                    </goals>
+                    <configuration>
+                        <sources>
+                            <source>${project.parent.basedir}/src/main/java</source>
+                        </sources>
+                    </configuration>
+                </execution>
```

Comment:
Shouldn't be necessary. Maven convention states that this directory is automatically added as a source location.